content/en/docs/instrumentation/cpp/exporters.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Exporters
+weight: 4
+---
+
+In order to visualize and analyze your
+[traces](/docs/concepts/signals/traces/#tracing-in-opentelemetry) and metrics,
+you will need to export them to a backend.
+
+## Logging exporter
+
+The logging exporter is useful for development and debugging tasks, and is the
+simplest to set up.
+
+```cpp
+auto ostream_exporter =
+    std::unique_ptr<opentelemetry::sdk::trace::SpanExporter>(new opentelemetry::exporter::trace::OStreamSpanExporter);
+```
+
+## OTLP endpoint
+
+To send trace data to an OTLP endpoint (like the [collector](/docs/collector) or
+Jaeger) you'll want to configure an OTLP exporter that sends to your endpoint.
+
+```cpp
+opentelemetry::exporter::otlp::OtlpHttpExporterOptions opts;
+opts.url = "http://localhost:4318/v1/traces";
+auto otlp_http_exporter =
+    std::unique_ptr<opentelemetry::sdk::trace::SpanExporter>(new opentelemetry::exporter::otlp::OtlpHttpExporter(opts));
+```
```

Comment:
nit - should we add link to https://github.com/open-telemetry/opentelemetry-cpp/blob/main/examples/otlp/README.md ?

Reply:
done.